Makefile

```diff
@@ -73,7 +73,7 @@ test: ## Run the tests
 	${DOCKER_COMPOSE} run --rm app npm run test
 
 run:
-	${DOCKER_COMPOSE} up --abort-on-container-exit --exit-code-from app; ${DOCKER_COMPOSE} down
+	${DOCKER_COMPOSE} up --abort-on-container-exit --exit-code-from app; exit=$$?; ${DOCKER_COMPOSE} down; exit $$exit
```

Comment:
Ctrl+C on a crashed application wrapped by `nodemon` gives me 130 as the `make` command level, but I when I echo the exit code it's 0. Confused, investigating.

Reply:
https://www.gnu.org/software/make/manual/make.html#index-exit-status-of-make states the exit status of make is always 0, 1 or 2; hence I think the Ctrl+C is propagated down to the process, then it's the (my) shell returning 130 (which is a [standard](https://www.tldp.org/LDP/abs/html/exitcodes.html) unrelated to make.

Reply:
Tested by adding `command: sh -c 'exit 42'` to `docker-compose.prod.yml`, and an echo before `exit`:
```
$ make prod
...
docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.prod.yml up --abort-on-container-exit --exit-code-from app; exit=$?; docker-compose --file .docker/docker-compose.yml --file .docker/docker-compose.prod.yml down; echo "exit=$exit"; exit $exit
Creating network "docker_default" with the default driver
Creating docker_app_1 ... done
Creating docker_console_1 ... done
Attaching to docker_app_1, docker_console_1
docker_app_1 exited with code 42
Aborting on container exit...
Stopping docker_console_1 ... done
Removing docker_console_1 ... done
Removing docker_app_1     ... done
Removing network docker_default
exit=42
Makefile:76: recipe for target 'run' failed
make[1]: *** [run] Error 42
make[1]: Leaving directory '/home/giorgio/code/article-store'
Makefile:85: recipe for target 'prod' failed
make: *** [prod] Error 2
[09:20:34][giorgio@Newton:~/code/article-store]$ echo $?
2
```

I'll spare everyone the details, but it's the `make run` command that first turns 42 into 2, whereas the parent `make prod` sees 2 as an error and also exits with 2. All expected behavior.